src/components/TodoWrapper.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from "react";
+import type { ChangeEvent } from "react";
+import { Todo } from "./Todo";
+import TodoForm from "./TodoForm";
+import { EditTodoForm } from "./EditForm";
+import type { TodoItem } from "../types/todo.types";
+
+export default function TodoWrapper() {
+  const [todos, setTodos] = useState<TodoItem[]>([]);
+  const [searchTerm, setSearchTerm] = useState<string>("");
```

Comment:
Here, we can allow typescript to do type inference.

```suggestion
  const [searchTerm, setSearchTerm] = useState("");
```

Reply:
fixed https://github.com/Sakshi-Simform/todo-list/commit/022d0f7d2b7c531e825c5797b4955f43941370a3